core/trino-spi/src/main/java/io/trino/spi/metrics/Distribution.java

```diff
@@ -23,4 +23,6 @@ public interface Distribution<T>
     double getMax();
 
     double getPercentile(double percentile);
```

Comment:
Can you get rid of `double getPercentile(double percentile)` ? This is just used in tests now